src/cgame/cg_main.c

```diff
@@ -2680,6 +2680,52 @@ void CG_LoadHudMenu(void)
 	CG_Text_SetActiveFont(0);
 }
 
+/**
+ * @brief
+ * Shim to load a path in a compatibility subdir first, before falling back to
+ * passed path.
+ */
+int CG_LoadCompatSource(const char *filename)
+{
+	if (cg.demoBackwardsCompat.compatPath[0] != '\0')
+	{
+		int res = trap_PC_LoadSource(va("%s%c%s", cg.demoBackwardsCompat.compatPath, PATH_SEP, filename));
+		if (res != 0)
+		{
+			if (developer.integer)
+			{
+				Com_Printf("CG_LoadCompatSource: %s\n", va("%s%c%s", cg.demoBackwardsCompat.compatPath, PATH_SEP, filename));
```

Comment:
In cgame, you'd usually use `CG_Printf` for printing to console (same in `CG_FOpenCompatFile`) as that is a cgame-specific print.